<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client_Add_Service</class>
 <widget class="QDialog" name="Client_Add_Service">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Adding Service to Car!</string>
  </property>
  <widget class="QPushButton" name="btn_Exit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>600</y>
     <width>91</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_PartsCost">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>102</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Parts Cost</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_totlaServiceCost">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Service Total</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Add_Parts_Used_In_The_Service">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>141</width>
     <height>34</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Add Parts</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Emoticon">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>590</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tbl_Parts_Used_In_Service">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>631</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_PartsUsed">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>361</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Parts Used on this Service. Double click to Remove a Part:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_Paid">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>560</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Service Paid?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Update_Service_Description">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>151</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Description</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_Servicedescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>631</width>
     <height>251</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Save_Hours_Worked">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>510</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Spin_Hand_Work_Hours">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>510</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_InsertWorkCost">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hours Worked:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_ServiceShortDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Service Hour Cost</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_Hour_cost">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_Parts_Cost">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_Service_Total">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>590</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <zorder>tbl_Parts_Used_In_Service</zorder>
  <zorder>txt_Servicedescription</zorder>
  <zorder>btn_Exit</zorder>
  <zorder>lbl_PartsCost</zorder>
  <zorder>lbl_totlaServiceCost</zorder>
  <zorder>btn_Add_Parts_Used_In_The_Service</zorder>
  <zorder>lbl_Emoticon</zorder>
  <zorder>lbl_PartsUsed</zorder>
  <zorder>check_Paid</zorder>
  <zorder>btn_Update_Service_Description</zorder>
  <zorder>btn_Save_Hours_Worked</zorder>
  <zorder>Spin_Hand_Work_Hours</zorder>
  <zorder>lbl_InsertWorkCost</zorder>
  <zorder>line_ServiceShortDescription</zorder>
  <zorder>label</zorder>
  <zorder>lcd_Hour_cost</zorder>
  <zorder>lcd_Parts_Cost</zorder>
  <zorder>lcd_Service_Total</zorder>
 </widget>
 <tabstops>
  <tabstop>btn_Add_Parts_Used_In_The_Service</tabstop>
  <tabstop>btn_Exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
